src/fastdiff.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/fastdiff
+ */
+
+/**
+ * Finds position of the first and last change in the given strings and generates set of changes. Set of changes
+ * can be applied to the input text in order to transform it into the output text, for example:
+ *
+ *		let input = '12abc3';
```

Comment:
I'd add at least two more, basic examples. One example: adding a few consecutive characters somewhere in the string. Second example: removing a few consecutive characters from the string. The example you provided is actually kind of an edge case, where characters were removed from the beginning and from the end. So, the output might actually be surprising for someone.

Other than that, I am not sure about the "usage" snippet. I have a feeling that the example is a bit uninspiring. Even though it shows how to use the output, somebody would think "why would I want to do it?". I have a feeling that we might want to drop this snippet.

I'd also shorten the examples:
```
* (...) for example:
*
*		fastDiff( '12a',	'12xyza' );	// [ { ... } ]
*		fastDiff( '12a',	'12aa' );	// [ { ... } ]
*		fastDiff( '12xyza',	'12a' );	// [ { ... } ]
*		fastDiff( '12aa',	'12a' );	// [ { ... } ]
```

Reply:
If the output is too long for the comment in the same line, it can be above or below starting with "Following produces:" and with extra line between each example.